src/style.css

```diff
@@ -2,2 +1,72 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+}
+
+.site-header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  height: 70px;
+  background-color: #fff;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.logo img {
+  height: 50px;
+  width: auto;
+  display: block;
+  margin-left: 20px;
+}
+
+.main-nav {
+  display: block;
+}
+
+.nav-list {
+  display: flex;
+  list-style: none;
   margin: 0;
+  padding: 0;
+}
+
+.nav-item {
+  text-decoration: none;
+  color: black;
+  margin-right: 20px;
+  height: 70px;
+  line-height: 70px;
+  display: inline-block;
+  position: relative;
+  text-align: center;
+}
+
+.nav-item:first-child {
+  margin-left: 0;
+}
+
+.nav-item:last-child {
+  margin-right: 0;
+}
+
+.nav-item:hover {
+  color: #00acdc;
+}
+
+.nav-item.is-active {
+  color: #00acdc;
+}
+
+.nav-item.is-active::after {
+  content: '';
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  width: 100%;
+  height: 4px;
+  background-color: #00acdc;
```

Comment:
The color `#00acdc` is used directly. Consider defining it as a CSS variable for consistency and maintainability as per the task requirements.